dataactvalidator/scripts/loaderUtils.py

```diff
@@ -5,12 +5,12 @@
 def clean_col_names(field):
     """Define some data-munging functions that can be applied to pandas
     dataframes as necessary"""
-    return str(field).lower().strip().replace(" ","_").replace(",","_")
+    return str(field).lower().strip().replace(" ", "_").replace(",", "_")
 
 
 class LoaderUtils:
     @classmethod
-    def padFunction(self, field, padTo, keepNull):
+    def padFunction(cls, field, padTo, keepNull):
```

Comment:
I think you said you'll do another round in a bit, but if we're not using the `cls` argument, we can turn this into a `staticmethod` (or just move it out of the class altogether)